src/SimpleReplay/replay.py

```diff
@@ -393,11 +397,21 @@ def save_query_stats(self, starttime, endtime, xid, query_idx):
                 )
 
     def get_tagged_sql(self, query_text, idx, transaction, connection):
-        json_tags = {
-            "xid": transaction.xid,
-            "query_idx": idx,
-            "replay_start": g_replay_timestamp.isoformat(),
-        }
+        if g_config["source_tag"]:
+            json_tags = {
+                "xid": transaction.xid,
+                "query_idx": idx,
+                "replay_start": g_replay_timestamp.isoformat(),
+                "source": g_config.get(["source_tag"]),
+            }
+        else:
+            json_tags = {
+                "xid": transaction.xid,
+                "query_idx": idx,
+                "replay_start": g_replay_timestamp.isoformat(),
+                "source": "SimpleReplay",
+            }
```

Comment:
This can probably be simplified to:

```
json_tags = {
                "xid": transaction.xid,
                "query_idx": idx,
                "replay_start": g_replay_timestamp.isoformat(),
                "source": g_config.get('source_tag', 'SimpleReplay'),
            }
```